client/components/CurrentWord.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import gameActions from '../store/actions/game';
+
+const useStyles = makeStyles((theme) => ({
+  word: {
+    fontSize: 14,
+    textAlign: 'center',
+    wordBreak: 'break-all',
+    [theme.breakpoints.up('sm')]: {
+      fontSize: 24,
+    },
+  },
+  character: {
+    marginRight: 4,
+    [theme.breakpoints.up('sm')]: {
+      marginRight: 6,
+    },
+    '&:last-child': {
+      marginRight: 0,
+    },
+  },
+  characterClickable: {
+    cursor: 'pointer',
+    '&:hover': {
+      outlineColor: theme.palette.text.primary,
+      outlineWidth: 1,
+      outlineStyle: 'dashed',
+    },
+  },
+  characterRevealed: {
+    textDecoration: 'underline',
+    fontWeight: 'bold',
+  },
+}));
+
+const CurrentWord = ({ selectedWord, currentWord, revealCharacter }) => {
```

Comment:
I'm confused about the difference between |selectedWord| and |currentWord|

Reply:
Yeah that was unfortunate naming, renamed it now to match the server side.
`currentWord` now contains the cleartext word if it's your turn to draw or if you've guessed it correctly
`currentWordEncoded` contains the encoded form, as in, `_ _ _ a _ _` etc.